lib/pure/terminal.nim

```diff
@@ -542,7 +542,8 @@ type
     fgBlue,                ## blue
     fgMagenta,             ## magenta
     fgCyan,                ## cyan
-    fgWhite                ## white
+    fgWhite,               ## white
+    fgDefault = 39         ## default terminal foreground color
```

Comment:
This seems to turn the enum into an enum with holes and so would theoretically break code. Not that it matters much.

Reply:
I force pushed a fix.. hopefully fixing it

Reply:
Looks like it still fails.. looks like the fix is to assign the 38/48 enums? Looking at ANSI reference, they should perhaps be named `fgTrue` and `bgTrue`.